source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -1034,17 +1034,26 @@ Mainnet
 Testnet
 -------
 
-    November 28, 2023
+    February 20, 2024
 
-    Version 6.2.3 removes the V1 gRPC API. This removes the configuration options ``CONCORDIUM_NODE_RPC_SERVER_PORT``, ``CONCORDIUM_NODE_RPC_SERVER_ADDRESS``, ``CONCORDIUM_NODE_RPC_SERVER_TOKEN``, ``CONCORDIUM_NODE_DISABLE_RPC_SERVER_NODE_ENDPOINTS`` and their command line equivalents. An additional health-check service was also added to the V2 GRPC API. This service conforms to the `standard GRPC health service API <https://github.com/grpc/grpc-proto/blob/master/grpc/health/v1/health.proto>`__.
+    The 6.3.0 release contains the following fixes and improvements:
 
-    As part of the tokenomics changes the node has new configuration options that use the new terminology. The existing options using the legacy terminology are still supported, however they are hidden.
+    - Fixed a bug where ``GetBlockPendingUpdates`` fails to report pending updates to the finalization committee parameters.
+    - GRPC queries are now run in dedicated threads. This improves node resource management and increases responsiveness of the GRPC server in cases of high number of concurrent queries. To support this a new option ``--grpc2-max-threads`` (environment variable ``CONCORDIUM_NODE_GRPC2_MAX_THREADS``) has been added, which specifies the number of threads that the node should use for processing gRPC requests. If not set this defaults to the number of (logical) CPUs.
+    - The option ``--grpc2-max-concurrent-streams`` now defaults to 200 from the previous unbounded value. This makes the node defaults safer.
+    - The GHC version has been updated to 9.6.4 (lts-22.9).
```

Comment:
I would remove this line, since users do not care about this, and don't even know what GHC is.

@mh-concordium If you confirm reduced startup time on the different platforms, we should add this here. We don't have to give specific numbers.

Reply:
Yes, looking at my measurements I can confirm reduced startup time compared to the previous released version. Tests were done on Mainnet.

Reply:
Should we state anything more specific about startup time other than it's reduced? E.g. 2 to 3 times faster or something.

Reply:
Looking at my measurements, I would say it is at least 2 times faster compared to the previous release on all distributions. However, I want to mention that this might not come true for all the users in production since there are multiple factors to be considered like performance of their machines, how they do measurements and if they are running multiple other programs while measuring the node startup time.

Reply:
I get a bit less difference on my workstation.

```
2024-02-19T13:32:59.424915131Z: INFO: Starting up the consensus layer
2024-02-19T13:35:17.282673130Z: INFO: Consensus layer started
```

vs.

```
2024-02-19T13:39:28.173492895Z: INFO: Starting up the consensus layer
2024-02-19T13:41:16.442668518Z: INFO: Consensus layer started
```
for the new one.

So I think we should just say improved startup time, the exact improvement depends on the environment.

Reply:
Ok. Sounds good to me.